templates/winthrop_staging/local_settings.py

```diff
@@ -0,0 +1,119 @@
+# This file is exec'd from settings.py, so it has access to and can
```

Comment:
Can templates be shared or included? i.e. would it make sense to have common localsettings snippets that are reused?

Reply:
Yes, they can be, and that might be a decent way to get around variables for multiline stanzas, now that you mention it.